pkg/accelerator/accelerator.go

```diff
@@ -0,0 +1,84 @@
+//
+// Use and distribution licensed under the Apache license version 2.
+//
+// See the COPYING file in the root project directory for full text.
+//
+
+package accelerator
+
+import (
+	"fmt"
+
+	"github.com/jaypipes/ghw/pkg/context"
+	"github.com/jaypipes/ghw/pkg/marshal"
+	"github.com/jaypipes/ghw/pkg/option"
+	"github.com/jaypipes/ghw/pkg/pci"
+)
+
+type AcceleratorDevice struct {
+	// the PCI address where the accelerator device can be found
+	Address string `json:"address"`
+	// pointer to a PCIDevice struct that describes the vendor and product
+	// model, etc
+	PCIDevice *pci.Device `json:"pci_device"`
+}
+
+func (dev *AcceleratorDevice) String() string {
+	deviceStr := dev.Address
+	if dev.PCIDevice != nil {
+		deviceStr = dev.PCIDevice.String()
+	}
+	nodeStr := ""
+	return fmt.Sprintf(
+		"device %s@%s",
+		nodeStr,
+		deviceStr,
+	)
+}
+
+type Info struct {
+	ctx                *context.Context
+	AcceleratorDevices []*AcceleratorDevice `json:"devices"`
```

Comment:
One tiny request... let's call the field `Devices` instead of `AcceleratorDevices` since the "Accelerator" part is assumed.